logbook-json/src/main/java/org/zalando/logbook/json/JacksonJsonFieldBodyFilter.java

```diff
@@ -47,17 +47,16 @@ public String filter(@Nullable final String contentType, final String body) {
 
     public String filter(final String body) {
         try {
```

Comment:
Could probably now combine the two try-catch blocks.

Reply:
Another observation: the tests fail because you call `writer.toString();`, but `generator` wasn't flushed yet. It probably make sense to still keep 2 try-cache blocks, but to move the `writer` creation to the outer one, and the return statement after the inner try-catch block if closed.

Reply:
@kasmarian 

Thanks for the feedback! 
I've reordered the 'try' blocks and fixed the unit tests.